src/compiler/types.ts

```diff
@@ -1007,6 +1013,10 @@ namespace ts {
         expression: LeftHandSideExpression;
         typeArguments?: NodeArray<TypeNode>;
     }
+    export interface SupportedExpressionWithTypeArguments extends ExpressionWithTypeArguments {

```

Comment:
Later: Yes, I think `EntityNameExpressionWithTypeArguments` is pretty good.

---
In reply to: [73190067](https://github.com/Microsoft/TypeScript/pull/10013#discussion_r73190067) [](ancestors = 73190067,73189548,72693405)